pkgs/snaplet/registry.yaml

```diff
@@ -0,0 +1,25 @@
+packages:
+  - name: snaplet
+    type: http
+    description: Seed your PostgreSQL database with production-like data so you can code, debug, and test with ease.
+    link: https://www.snaplet.dev/
+    url: https://snaplet-public.s3-accelerate.amazonaws.com/cli/beta/snaplet-{{.OS}}-{{.Arch}}-{{.Version}}
+    overrides:
+      - goos: windows
+        url: https://snaplet-public.s3-accelerate.amazonaws.com/cli/beta/snaplet-{{.OS}}-{{.Arch}}-{{.Version}}.exe
```

Comment:
```suggestion
```

This override isn't needed because aqua completes the file extension `.exe` by default on Windows.

Reply:
Ah good stuff 👍